handlers/k8s.go

```diff
@@ -0,0 +1,47 @@
+package handlers
+
+import (
+	"errors"
+	"time"
+
+	"github.com/deis/k8s-claimer/k8s"
+	"github.com/deis/k8s-claimer/leases"
+	"k8s.io/kubernetes/pkg/api"
+)
+
+var (
+	errNoExpiredLeases = errors.New("no expired leases exist")
+)
+
+// findExpiredLease searches in the leases in the svc annotations and returns the cluster name of
```

Comment:
I assume this is a convenience function for consumers that need to ensure no expired leases exist? In any event, we might want to over-explain the purpose of returning the "first found" expired lease, as this begs the question of why aren't we prioritizing really stale expired leases over recent ones.